sql-prepare-plan-cache.md

```diff
@@ -5,16 +5,12 @@ summary: Learn about SQL Prepare Execution Plan Cache in TiDB.
 
 # SQL Prepare Execution Plan Cache
 
-TiDB supports execution plan caching for `Prepare` / `Execute` queries.
+Starting with TiDB 5.0, the execution plan of prepared statements is cached by default. This includes both forms of prepared statements:
```

Comment:
I don't think it's good to document the features of the upcoming version.
Reference: https://developers.google.cn/style/future.

Reply:
How would you word it? The change applies to master, but is not cherry-picked. It is useful for a user to know that this is a new behavior.

Reply:
What about "In the latest development version of TiDB, ..."? And mark the PR as `follow-up`. When any 5.0 version is released, we can change the wording to "Starting with TiDB 5.0". Is it OK?

Reply:
Sounds good to me. I'll do that.